apps/vs-code-designer/src/app/utils/codeless/connection.ts

```diff
@@ -235,11 +244,17 @@ export async function getConnectionsAndSettingsToUpdate(
         settingsToAdd,
         parametersFromDefinition
       );
+
```

Comment:
NIT: Why don't we just simply show the information message here? Instead of relying on a variable? since we don't use the variable anywhere else?

Reply:
Agreed no change needed as variable in a for loop which checks if any key is refreshed.